File: PRAK401-2210817120011-MAULIDASARI.c
#include <stdio.h>

int main() {
int n,i,j;
char A;
   
   printf("Masukkan Angka Kelipatan : ");
   scanf("%d %c \n", &n, &A);    
  
for(i=1; i<=50; i++){
    if(i%n!= 0){
    printf("%d", i);
    }
    printf("", i);
    if(i%n==0){
        printf("%c", A);
    }
    printf(" ", A);
    printf("\n");
   }
}

File: PRAK401-2210817120011-MAULIDASARI.py

n= int(input("Masukkan Angka Kelipatan : "))
simbol=input("Masukkan Simbol : ")
     
for i in range (1, 51):
    if i % n == 0 :
        print(simbol,end=' ')
    else:
        print(i, end=' ')

